admin/includes/template/partials/tplAdminLeadItemRows.php

```diff
@@ -32,4 +32,16 @@
         <?php } ?>
     </tr>
 <?php } ?>
+<?php if ($tplVars['listingBox']['formattedTotals'] != null) { ?>
+<tr>
+<?php $cols = count($tplVars['leadDefinition']['listMap']) + 1; ?>
+<td colspan="<?php echo $cols;?>"><?php zen_draw_separator(); ?></td>
+</tr>
+    <tr>
+        <td>&nbsp;</td> <!-- account for checkbox --> 

```

Comment:
Given this column always exists, could we put a title in here 
e.g. Page Total instead of &nbsp; 
I've tried this and it does make the layout look a bit weird but that's a css issue, and we can fix that later